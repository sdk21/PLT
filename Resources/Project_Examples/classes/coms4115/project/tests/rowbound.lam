matrix A = { 0, 0; 0, 0 };
print A[2, 0];
