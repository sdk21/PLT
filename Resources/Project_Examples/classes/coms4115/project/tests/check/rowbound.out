Runtime error: Index out of bounds!
